pkg/storage/secret/encryption/encrypted_value_store.go

```diff
@@ -86,45 +110,63 @@ func (s *encryptedValStorage) Update(ctx context.Context, namespace string, uid
 }
 
 func (s *encryptedValStorage) Get(ctx context.Context, namespace string, uid string) (*contracts.EncryptedValue, error) {
-	encryptedValueRow := &EncryptedValue{
-		UID:       uid,
-		Namespace: namespace,
+	req := &readEncryptedValue{
+		SQLTemplate: sqltemplate.New(s.dialect),
+		Namespace:   namespace,
+		UID:         uid,
+	}
+	query, err := sqltemplate.Execute(sqlEncryptedValueRead, req)
+	if err != nil {
+		return nil, fmt.Errorf("execute template %q: %w", sqlEncryptedValueRead.Name(), err)
 	}
 
-	err := s.db.WithDbSession(ctx, func(sess *sqlstore.DBSession) error {
-		found, err := sess.Get(encryptedValueRow)
-		if err != nil {
-			return fmt.Errorf("could not get row: %w", err)
-		}
+	var encryptedValue *EncryptedValue
+	res, err := s.db.GetSqlxSession().Query(ctx, query, req.GetArgs()...)
+	if err != nil {
+		return nil, fmt.Errorf("getting row: %w", err)
+	}
 
-		if !found {
-			return ErrEncryptedValueNotFound
+	defer func() { _ = res.Close() }()
+
+	if res.Next() {
+		row := &EncryptedValue{}
```

Comment:
Nit: you can add `encryptedValue := EncryptedValue{}` and then on the scan do `&encryptedValue.UID` etc, no?